### DISCUSSION TITLE
Working with templates

### DISCUSSION THREAD

**Discussion by SergiuszKotecki** [django-money/django-money#643] [Q&A]
Hello all!

Usually when I am building a custom form all what I have to do is a put {{form.money_field.html_name}} into input tags. But this time case is a little bit different.  Mainly because {{money_field}} returns input and select tags. So the question is, how to separate them to achieve such effect:

```
<input name={{ form.money_field.html_name? }} >
<select name="pets" id="pet-select">
{% for currency in money_field.?someattribute" %}
<option value="{{currency }}"> {{currency }}</option>
{%endfor %}

</select>
```

**Comment by alexandremjacques** [comment]
Does anybody has an input on this? I'm trying to layout my form using the `as_crispy_field` from crispy-forms and having the same issue.

Any help would be appreciated.

TIA

**Reply by benjaoming** [reply]
I can't come up with an example or really test this. The `MoneyField` uses `MoneyWidget` which inherits from Django's `MultiWidget`.

I think you should be able to iterate over the widgets in your template. Something like `{% for subfield in your_form.money_field %}` or {{ your_form.money_field.0 }} and {{ your_form.money_field.1 }} ?

**Reply by alexandremjacques** [reply]
Thanks. I'll give it a try.

**Reply by alexandremjacques** [reply]
Unfortunately this is not a solution. `{{ your_form.money_field.0 }}` brings the whole MultiWidget instead for the currency or the monetary value of it.
